### ISSUE TITLE
Validation Issue

### ISSUE THREAD

**Issue by hamzaious** [p32929/AndroidAppLockscreen#4]
There is no callback for validation of password, which means how we can confirm about wrong credentials, 
EasyLock.checkPassword(this); return "void" it should return a boolean ??

**Comment by p32929** [comment]
I agree. I'll update it. Or if you wanna make a pull request, feel free to. thanks
